### DISCUSSION TITLE
I am not able to generate .tsv output file after running SWATH DIA wiff files in DIA-NN

### DISCUSSION THREAD

**Discussion by NamanKharbanda1997** [vdemichev/DiaNN#696] [Q&A]
**This is what the output looks like after searching the files. I am not getting the output  .tsv files in output folder**

Compiled on Apr 14 2022 15:31:19
Current date and time: Mon May  1 17:38:08 2023
CPU: GenuineIntel Intel(R) Xeon(R) W-2245 CPU @ 3.90GHz
SIMD instructions: AVX AVX2 AVX512CD AVX512F FMA SSE4.1 SSE4.2 
Logical CPU cores: 8
Thread number set to 8
Output will be filtered at 0.01 FDR
Precursor/protein x samples expression level matrices will be saved along with the main report
A spectral library will be generated
Deep learning will be used to generate a new in silico spectral library from peptides provided
Library-free search enabled
Min fragment m/z set to 200
Max fragment m/z set to 1800
N-terminal methionine excision enabled
In silico digest will involve cuts at K*,R*
Maximum number of missed cleavages set to 1
Min peptide length set to 7
Max peptide length set to 30
Min precursor m/z set to 300
Max precursor m/z set to 1800
Min precursor charge set to 1
Max precursor charge set to 4
Cysteine carbamidomethylation enabled as a fixed modification
A spectral library will be created from the DIA runs and used to reanalyse them; .quant files will only be saved to disk during the first step
Highly heuristic protein grouping will be used, to reduce the number of protein groups obtained; this mode is recommended for benchmarking protein ID numbers; use with caution for anything else
When generating a spectral library, in silico predicted spectra will be retained if deemed more reliable than experimental ones
Fixed-width center of each elution peak will be used for quantification
Interference removal from fragment elution curves disabled
DIA-NN will optimise the mass accuracy automatically using the first run in the experiment. This is useful primarily for quick initial analyses, when it is not yet known which mass accuracy setting works best for a particular acquisition scheme.
Exclusion of fragments shared between heavy and light peptides from quantification is not supported in FASTA digest mode - disabled; to enable, generate an in silico predicted spectral library and analyse with this library
WARNING: MBR turned off, two or more raw files are required

1 files will be processed
[0:00] Loading FASTA C:\Users\ZenoTOF\Downloads\uniprot-download_true_format_fasta_includeIsoform_true_query__28Huma-2023.04.20-11.33.09.24.fasta
[0:12] Processing FASTA
[0:28] Assembling elution groups
[0:45] 8326594 precursors generated
[0:45] Gene names missing for some isoforms
[0:45] Library contains 51462 proteins, and 31013 genes
[0:46] Encoding peptides for spectra and RTs prediction
[1:02] Predicting spectra and IMs
[59:02] Predicting RTs
[73:06] Decoding predicted spectra and IMs
[73:29] Decoding RTs
[73:35] Saving the library to D:\SCIEX OS Data\Default\Data\TKMLAB\2023_05_01\OUTPUT_REPORT.predicted.speclib
[73:45] Initialising library

[73:50] File #1/1
[73:50] Loading run D:\SCIEX OS Data\Default\Data\TKMLAB\2023_05_01\HFF98_SWATH_Swati.wiff

DIA-NN exited
DIA-NN-plotter.exe "D:\SCIEX OS Data\Default\Data\TKMLAB\2023_05_01\1report.stats.tsv" "D:\SCIEX OS Data\Default\Data\TKMLAB\2023_05_01\1report.tsv" "D:\SCIEX OS Data\Default\Data\TKMLAB\2023_05_01\1report.pdf"
PDF report will be generated in the background

**Comment by vdemichev** [comment]
Likely Sciex DLLs not copied, please see DIA-NN docs on this.
